rpc/eth_api.go

```diff
@@ -539,6 +539,14 @@ func (api *ethAPI) FeeHistory(
 	return w3c.Eth.FeeHistory(blockCount, lastBlock, rewardPercentiles)
 }
 
+// GetBalance returns the amount of wei for the given address in the state of the
+// given block number.
+func (api *ethAPI) GetProof(ctx context.Context, addr common.Address, storageKeys []string, blockNrOrHash web3Types.BlockNumberOrHash) (*web3Types.AccountResult, error) {
+	w3c := GetEthClientFromContext(ctx)
+	api.inputBlockMetric.Update2(&blockNrOrHash, "eth_getProof", w3c.Eth)
```

Comment:
What's the difference between `Update1` and `Update2`?

Reply:
`Update1` accepts `BlockNumber`, `Update2` accepts `BlockNumberOrHash`